en/guide/dronecore_extensions.md

```diff
@@ -65,7 +64,11 @@ Plugins in extension libraries are exactly the same as "normal" DroneCore plugin
 
 ## Testing
 
-Tests in extension libraries are written and run exactly the same as "normal" DroneCore plugin tests.
+Tests in extension libraries are written in exactly the same as "normal" DroneCore plugin tests.
+
+The only difference is that all external tests should be compiled into their own "test runner". 
+The test runner for the "external example" is defined in 
+[/integration_tests/CMakeLists.txt]( https://github.com/dronecore/DroneCore/blob/{{ book.github_branch }}/external_example/integration_tests/CMakeLists.txt) (named `external_example_integration_tests_runner`).

```

Comment:
So as above,I can see how test runner is created, but not how it "should" be launched.

Reply:
`make && build/default/integration_tests/integration_tests_runner`